src/test/bassTranslatorTest.js

```diff
@@ -0,0 +1,189 @@
+function mod(a, b) {
```

Comment:
"Mod" have been already defined in Utils.js why not to use it?

Reply:
I'll create proper tests in other branch